test/sys/test_event.rs

```diff
@@ -11,7 +11,7 @@ fn test_struct_kevent() {
     let actual = KEvent::new(
         0xdead_beef,
         EventFilter::EVFILT_READ,
```

Comment:
@asomers, any idea what we should do to things like this `enum EventFilter`? 

```
use nix::sys::event::{EvFlags, EventFilter};
```

Honestly, it feels weird to me when putting them together 🥲, one aims to be short, typical C-style, one aims to be self-explanatory.

Reply:
If does seem odd now. I think they'll need hard to remember like this. We should probably make the two names more consistent.

Reply:
Yeah